includes/definitions/discovery/f5.yaml

```diff
@@ -20,3 +21,94 @@ modules:
                 total: F5-BIGIP-SYSTEM-MIB::sysGlobalHostSwapTotal
                 descr: 'Swap Memory'
                 index: 2
+    sensors:
+        state:
+            data:
+                -
+                    oid: sysCmFailoverStatusId
+                    num_oid: '.1.3.6.1.4.1.3375.2.1.14.3.1.{{ $index }}'
+                    index: 'sysCmFailoverStatusId.{{ $index }}'
+                    descr: Failover State
+                    group: HA
+                    states:
+                        - { value: 0, descr: 'Unknown', graph: 0, generic: 3 }
+                        - { value: 1, descr: 'OffLine', graph: 0, generic: 2 }
+                        - { value: 2, descr: 'ForcedOffline', graph: 0, generic: 2 }
+                        - { value: 3, descr: 'Standby', graph: 0, generic: 0 }
+                        - { value: 4, descr: 'Active', graph: 0, generic: 0 }
+                -
+                    oid: sysCmSyncStatusId
+                    num_oid: '.1.3.6.1.4.1.3375.2.1.14.1.1.{{ $index }}'
+                    index: 'sysCmSyncStatusId.{{ $index }}'
+                    descr: Sync State
+                    group: HA
+                    states:
+                        - { value: 0, descr: 'unknown', graph: 0, generic: 3 }
+                        - { value: 1, descr: 'syncing', graph: 0, generic: 1 }
+                        - { value: 2, descr: 'needManualSync', graph: 0, generic: 2 }
+                        - { value: 3, descr: 'inSync', graph: 0, generic: 0 }
+                        - { value: 4, descr: 'syncFailed', graph: 0, generic: 1 }
+                        - { value: 5, descr: 'syncDisconnected', graph: 0, generic: 2 }
+                        - { value: 6, descr: 'standalone', graph: 0, generic: 0 }
+                        - { value: 7, descr: 'awaitingInitialSync', graph: 0, generic: 1 }
+                        - { value: 8, descr: 'incompatibleVersion', graph: 0, generic: 2 }
+                        - { value: 9, descr: 'partialSync', graph: 0, generic: 2 }
+                -
+                    oid: sysCmFailoverStatusColor
+                    num_oid: '.1.3.6.1.4.1.3375.2.1.14.3.3.{{ $index }}'
+                    index: 'sysCmFailoverStatusColor.{{ $index }}'
+                    descr: Failover State Color
+                    group: HA
+                    states:
+                        - { value: 0, descr: 'Green: functioning correctly', graph: 0, generic: 0 }
+                        - { value: 1, descr: 'Yellow: functioning suboptimally', graph: 0, generic: 1 }
+                        - { value: 2, descr: 'Red: requires attention to function correctly', graph: 0, generic: 2 }
+                        - { value: 3, descr: 'Blue: status is unknown', graph: 0, generic: 3 }
+                        - { value: 4, descr: 'Gray: intentionally not functioning', graph: 0, generic: 0 }
```

Comment:
Nice work!

I don't know the Hardware but it feels for me that line 67+68 should better have generic 3.

The description would be cleaner without Color I think.

Reply:
The F5 Status Color is a bit confused.
But It's how It shows at the device. Take a look:
 
![image](https://github.com/librenms/librenms/assets/79482451/e055eee4-6ca6-4c41-bc36-92432c6dc33d)
bash-4.2$ snmpwalk -Os -c public -v 2c 10.10.10.41 .1.3.6.1.4.1.3375.2.1.14 | grep  "14.1.2\|14.3.1\|14.3.2\|14.3.3"
enterprises.3375.2.1.14.1.2.0 = STRING: "In Sync"
enterprises.3375.2.1.14.3.1.0 = INTEGER: 4
`- { value: 4, descr: 'Active', graph: 0, generic: 0 }`
enterprises.3375.2.1.14.3.2.0 = STRING: "ACTIVE"
enterprises.3375.2.1.14.3.3.0 = INTEGER: 0
`- { value: 0, descr: 'Green: functioning correctly', graph: 0, generic: 0 }`

----

![image](https://github.com/librenms/librenms/assets/79482451/3701b813-00e9-457f-9c8f-b50cdeb059b4)
bash-4.2$ snmpwalk -Os -c public -v 2c 10.10.10.42 .1.3.6.1.4.1.3375.2.1.14 | grep  "14.1.2\|14.3.1\|14.3.2\|14.3.3"
enterprises.3375.2.1.14.1.2.0 = STRING: "In Sync"
enterprises.3375.2.1.14.3.1.0 = INTEGER: 3
`- { value: 3, descr: 'Standby', graph: 0, generic: 0 }`
enterprises.3375.2.1.14.3.2.0 = STRING: "STANDBY"
enterprises.3375.2.1.14.3.3.0 = INTEGER: 4
`- { value: 4, descr: 'Gray: intentionally not functioning', graph: 0, generic: 0 }`

----

![image](https://github.com/librenms/librenms/assets/79482451/8416c777-727d-4fbe-96a8-78b8ed01af48)
bash-4.2$ snmpwalk -Os -c public -v 2c 10.10.10.42 .1.3.6.1.4.1.3375.2.1.14 | grep  "14.1.2\|14.3.1\|14.3.2\|14.3.3"
enterprises.3375.2.1.14.1.2.0 = STRING: "In Sync"
enterprises.3375.2.1.14.3.1.0 = INTEGER: 2
`- { value: 2, descr: 'ForcedOffline', graph: 0, generic: 2 }`
enterprises.3375.2.1.14.3.2.0 = STRING: "FORCED OFFLINE"
enterprises.3375.2.1.14.3.3.0 = INTEGER: 5
`- { value: 5, descr: 'Black: not connected to any peers', graph: 0, generic: 2 }`

Reply:
Thank you for the clear explanation.